ccm_web/client/src/components/Alert.tsx

```diff
@@ -15,16 +15,17 @@ const StyledGrid = styled(Grid)((
     theme
   }
```

Comment:
I see some linting waring saying theme this never used? This happens in 2 places in this file. Might need to remove it?

Reply:
I just committed a rewrite of the styles to make it a little more simplified, I realized I needed theme for a padding style that I forgot to add.